honeybee_schema/energy/schedule.py

```diff
@@ -239,6 +250,32 @@ class ScheduleRulesetAbridged(NamedEnergyBaseModel):
     )
 
 
+class ScheduleRuleset(ScheduleRulesetAbridged):
+    """Used to define a schedule for a default day, further described by ScheduleRule."""
+
+    type: constr(regex='^ScheduleRuleset$') = 'ScheduleRuleset'
+
+    day_schedules: List[ScheduleDay] = Field(
+        ...,
+        description='A list of ScheduleDays that are referenced in the other keys of '
+        'this ScheduleRuleset.'
+    )
+
+    schedule_rules: List[ScheduleRule] = Field(
```

Comment:
This should be a list of `ScheduleRuleAbridged`

Reply:
Actually, there's no need for you to add either the `schedule_rules` key or the `day_schedules` key since these are both the same between `ScheduleRuleset` and `ScheduleRulesetAbirdged` and you're already inheriting from `ScheduleRulesetAbridged`